src/shared/Button/index.lite.tsx

```diff
@@ -14,7 +14,24 @@ export default function Button(props: ButtonProps) {
       ref={props.buttonRef}
       type={props.type || 'button'}
       class={`${styles.btn}${state.variantCss}${props.classNames ? ' ' + props.classNames : ''}`}
+      disabled={props.isLoading}
       onClick={(event) => props.handleClick?.(event)}>
+      {props.isLoading ? (
+        <svg
+          xmlns='http://www.w3.org/2000/svg'
+          width='24'
+          height='24'
+          className={`${styles.loading}`}
+          viewBox='0 0 24 24'
+          fill='none'
+          stroke='currentColor'
+          stroke-width='2'
+          stroke-linecap='round'
+          stroke-linejoin='round'
+          class='lucide lucide-loader-2'>
```

Comment:
Do we need these classes?

Reply:
It's the loading svg, we can use from a icon library or you want me to create a custom one?

Reply:
Ahh, I could have been more specific here. So we already have a spinner here: https://github.com/boxyhq/ui/blob/148aff4a8de84a22c9e9d3095ae110126c03e2b3/src/shared/LoadingContainer/index.lite.tsx#L8. What you can do is create a new `Spinner` component (inside shared folder) by copying just this spinner div and its linked CSS. That way we can use it in both places.